code/utils.q

```diff
@@ -484,13 +500,14 @@ utils.getRelevantDates:{[dateInfo;allDates]
        ];
     '"startDate provided must be an individual date or regex string"
     ];
-  if[0=count relevantDates;'"No dates requested matched a presently saved model folder"];
+  if[0=count relevantDates;
+    '"No dates requested matched a presently saved model folder"];
```

Comment:
`]` should be on the next line for if statements style

Reply:
this is done